chef/cookbooks/neutron/recipes/common_agent.rb

```diff
@@ -257,10 +294,40 @@
         Chef::Log.info("#{name} usurped #{res[1].join(", ")} routes from #{bound_if}") unless res[1].empty?
       end
     end
+
+    execute "Neutron: add #{bound_if} to #{name}" do
+      command "ovs-vsctl add-port #{name} #{bound_if}"
+      not_if not_if_clause
+    end
     source = ::Nic.new(bound_if)
     # filter out cluster VIPs that a currently assigned to the interface
     addresses = source.addresses.select { |address| my_addresses.include? address }
     routes = source.routes
+
+    not_if_condition = addresses.empty? and routes.empty?
```

Comment:
Tiny detail, but it's slightly confusing to have `not_if_clause` and `not_if_condition`. Can we rename to `not_if_clause_before` and `not_if_clause_after`?